opensrp-anc/src/main/java/org/smartregister/anc/library/util/Utils.java

```diff
@@ -225,6 +224,10 @@ public static void proceedToContact(String baseEntityId, HashMap<String, String>
             ContactModel baseContactModel = new ContactModel();
             JSONObject form = baseContactModel.getFormAsJson(quickCheck.getFormName(), baseEntityId, locationId);
 
+            JSONObject globals = new JSONObject();
+            globals.put(ConstantsUtils.CONTACT_NO, personObjectClient.get(DBConstantsUtils.KeyUtils.NEXT_CONTACT));
+            form.put(ConstantsUtils.GLOBAL, globals);
+
```

Comment:
This is a stop gap we need to process quick check globals the same way the other forms globals are processed.  @bennsimon please create an issue of the remaining work and schedule it in

Reply:
https://github.com/OpenSRP/opensrp-client-anc/issues/512